manifest.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.gitlab.zehkira.Monophony
```

Comment:
it should be `io.gitlab`

Reply:
My [previously added app](https://github.com/flathub/com.gitlab.zehkira.Myuzi) uses `com.gitlab` though :thinking:

Reply:
The rules have changed since. (Or rather that we started to enforce some things.)

Reply:
https://docs.flatpak.org/en/latest/conventions.html#application-ids